### PR TITLE
Rebrand "Email" to "Professional Email"

### PR DESCRIPTION
**Pull request by daledupreez**
#### Changes proposed in this Pull Request

* This PR updates all client-side uses of "Email" referring to our Email product to be "Professional Email", which includes a number of changes to use our common product name function and string substitution, mostly so we have consistent translations for "Professional Email"
* Note that we still have server-defined strings that will appear in the UI, so this doesn't address all cases where "Email" may appear instead of "Professional Email".

#### Testing instructions

There are a number of locations where we need to check that the new brand is displayed:
* The card header in the email comparison page (shown when adding email to an existing domain).
* The terms of service displayed in checkout when buying a new Professional Email subscription or adding Professional Email mailboxes to an existing subscription.
* When opening up the external link to manage your Professional Email settings via the Titan Control Panel, ensure that the redirect page uses "Professional Email" before redirecting.
* The card header for the new email plan management page should be "Professional Email settings"
* The card header and section header for the iframed control panel should include "Professional Email" -- this can be enabled by adding `?flags=titan/iframe-control-panel` to your URL

##### New email plan page
Screenshot 2021-05-11 at 12 45 17

##### Control panel redirect page
Screenshot 2021-05-11 at 12 48 55

##### Terms of service in checkout
Screenshot 2021-05-11 at 12 47 35

##### Page header and section header for iframed control panel
Screenshot 2021-05-11 at 12 56 48

### CODE REVIEW

File: client/my-sites/email/email-providers-comparison/index.jsx [hunk 1]
```diff
@@ -401,7 +401,7 @@ class EmailProvidersComparison extends React.Component {
 			<EmailProviderCard
 				providerKey="titan"
 				logo={ logo }
-				title={ translate( 'Email' ) }
+				title={ translate( 'Professional Email' ) }
```
Comment (a8ci18n) [inline client/my-sites/email/email-providers-comparison/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **31** times:
`translate( 'Professional email address' )` *ES Score: 7*

File: client/my-sites/checkout/composite-checkout/components/titan-terms-of-service.jsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ function TitanTermsOfService( { cart, translate } ) {
 	}
 
 	const titanTerms = translate(
-		"You understand that your Email service will be powered by Titan and is subject to Titan's {{titanCustomerTos}}Customer Terms of Service{{/titanCustomerTos}}, {{titanAup}}Acceptable Use Policy{{/titanAup}}, and {{titanPrivacy}}Privacy Policy{{/titanPrivacy}}.",
+		"You understand that your Professional Email service will be powered by Titan and is subject to Titan's {{titanCustomerTos}}Customer Terms of Service{{/titanCustomerTos}}, {{titanAup}}Acceptable Use Policy{{/titanAup}}, and {{titanPrivacy}}Privacy Policy{{/titanPrivacy}}.",
```
Comment (a8ci18n) [inline client/my-sites/checkout/composite-checkout/components/titan-terms-of-service.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/email/email-management/titan-control-panel-redirect/index.jsx [hunk 3]
```diff
@@ -90,7 +90,7 @@ class TitanControlPanelRedirect extends React.Component {
 				<EmptyContent illustration="" title="">
 					<Card>
 						<Spinner size={ 40 } />
-						<h1>{ translate( 'Redirecting you to your Email Control Panel' ) }</h1>
+						<h1>{ translate( 'Redirecting you to your Professional Email Control Panel' ) }</h1>
```
Comment (a8ci18n) [inline client/my-sites/email/email-management/titan-control-panel-redirect/index.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/my-sites/email/titan-redirector/index.jsx [hunk 4]
```diff
@@ -115,7 +115,7 @@ class TitanRedirector extends Component {
 				<EmptyContent
 					title={ translate( "We couldn't locate your account details" ) }
 					line={ translate(
-						'Please check that you purchased the Email subscription, as only the original purchaser can manage billing and add more accounts'
+						'Please check that you purchased the Professional Email subscription, as only the original purchaser can manage billing and add more accounts'
```
Comment (a8ci18n) [inline client/my-sites/email/titan-redirector/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/lib/titan/get-titan-product-name.js [hunk 5]
```diff
@@ -1,18 +1,8 @@
...
+import { translate } from 'i18n-calypso';
 
 export function getTitanProductName() {
-	if ( isEnabled( 'titan/phase-2' ) ) {
-		return i18n.translate( 'Email', {
-			comment: 'Email is the name of a WordPress.com product, not just a capitalization of "email"',
-		} );
-	}
-	return i18n.translate( 'Titan Mail' );
+	return translate( 'Professional Email' );
```
Comment (a8ci18n) [inline client/lib/titan/get-titan-product-name.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **31** times:
`translate( 'Professional email address' )` *ES Score: 7*

Comment (stephanethomas) [reply client/lib/titan/get-titan-product-name.js]: I wonder if we shouldn't emphasis the fact that this is a product name, as this provides more context to translators.

File: client/my-sites/email/email-management/titan-control-panel-login-card/index.jsx [hunk 6]
```diff
@@ -118,7 +120,7 @@ class TitanControlPanelLoginCard extends React.Component {
 				productName: getTitanProductName(),
 			},
 			comment:
-				'%(domainName)s is a domain name, e.g. example.com; %(productName)s is the product name, either Email or Titan Mail',
+				'%(domainName)s is a domain name, e.g. example.com; %(productName)s is the product name, which should be Professional Email',
```
Comment (stephanethomas) [inline client/my-sites/email/email-management/titan-control-panel-login-card/index.jsx]: I think it's worth specifying everywhere that `%(productName)s` will be the translated version of `Professional Email`, so it's clear for translators what string they should expect.

File: client/my-sites/email/email-management/titan-management-iframe/index.jsx [hunk 7]
```diff
@@ -77,11 +78,25 @@ class TitanManagementIframe extends React.Component {
 					<QueryEmailAccounts siteId={ selectedSiteId } />
 				) }
 				<QuerySiteDomains siteId={ selectedSiteId } />
-				<DocumentHead title={ translate( 'Email Management' ) } />
+				<DocumentHead
+					title={ translate( '%(titanProductName)s Management', {
```
Comment (stephanethomas) [inline client/my-sites/email/email-management/titan-management-iframe/index.jsx]: Shouldn't we use the same title displayed in the header of that page (i.e `%(titanProductName)s settings`?

**Review by stephanethomas — APPROVED** [review]
This looks good (I had some suggestions that are not mandatory at all), and works fine.

File: client/lib/titan/get-titan-product-name.js [hunk 8]
```diff
@@ -1,18 +1,10 @@
...
+import { translate } from 'i18n-calypso';
 
 export function getTitanProductName() {
-	if ( isEnabled( 'titan/phase-2' ) ) {
-		return i18n.translate( 'Email', {
-			comment: 'Email is the name of a WordPress.com product, not just a capitalization of "email"',
-		} );
-	}
-	return i18n.translate( 'Titan Mail' );
+	return translate( 'Professional Email', {
```
Comment (a8ci18n) [inline client/lib/titan/get-titan-product-name.js:7]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **31** times:
`translate( 'Professional email address' )` *ES Score: 7*

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5906810

Thank you @daledupreez for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5906810).

**Pull request merged**